.github/workflows/extbuild.yml

```diff
@@ -71,31 +71,35 @@ jobs:
           parallelio_path: $HOME/pio
       - name: Cache CDEPS
         id: cache-cdeps
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
-          path: $HOME/cdeps
-          key: ${{ runner.os }}-${{ env.CDEPS_VERSION }}.cdeps
+          path: /homme/runner/work/CMEPS/CMEPS/build-cdeps
+          key: ${{ runner.os }}-${{ env.CDEPS_VERSION }}.cdeps1
           
       - name: checkout CDEPS
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: ESCOMP/CDEPS
           path: cdeps-src
           ref: ${{ env.CDEPS_VERSION }}
+      - name: get genf90
+        run: |
+          cd cdeps-src
+          git submodule update --init 
```

Comment:
Should EMC/CMEPS github workflows use EMC repos (particularly with this genf90 in ESCOMP/CDEPS)?

Reply:
I don't think this has anything to do w/ our fork.